Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: i18n/sources.txt i18n/messages.pot
```

Comment:
Comment for whole file: we haven't used `Makefile` until now because we want all commands which might need to be run in dev to be cross-platform including Windows.

Reply:
For a developer, extracting and POT generation will be so cumbersome and boring, just like dependencies updating, right? :stuck_out_tongue_winking_eye: 

So I think we should run that automatically.